scripts/exregional_get_verif_obs.sh

```diff
@@ -43,532 +43,44 @@ done
-# this script.
-#
-# Notes about the data and how it's used for verification:
-# 
-# 1. Accumulation is currently hardcoded to 01h. The verification will 
-# use MET/pcp-combine to sum 01h files into desired accumulations.
-#
-# 2. There is a problem with the valid time in the metadata for files
-# valid from 19 - 00 UTC (or files under the '00' directory). This is
-# accounted for in this script for data retrieved from HPSS, but if you
-# have manually staged data on disk you should be sure this is accouned
-# for. See in-line comments below for details.
-#
-#
-# MRMS (Multi-Radar Multi-Sensor) radar observations
-# ----------
-# If data is available on disk, it must be in the following 
-# directory structure and file name conventions expected by verification
-# tasks:
-#
-# {MRMS_OBS_DIR}/{YYYYMMDD}/[PREFIX]{YYYYMMDD}-{HH}0000.grib2,
-# 
-# Where [PREFIX] is MergedReflectivityQCComposite_00.50_ for reflectivity
-# data and EchoTop_18_00.50_ for echo top data. If data is not available
-# at the top of the hour, you should rename the file closest in time to
-# your hour(s) of interest to the above naming format. A script
-# "ush/mrms_pull_topofhour.py" is provided for this purpose. 
-#
-# If data is retrieved from HPSS, it will automatically staged by this
-# this script.
-#
-#
-# NDAS (NAM Data Assimilation System) conventional observations
-# ----------
-# If data is available on disk, it must be in the following 
-# directory structure and file name conventions expected by verification
-# tasks:
-#
-# {NDAS_OBS_DIR}/{YYYYMMDD}/prepbufr.ndas.{YYYYMMDDHH}
-# 
-# Note that data retrieved from HPSS and other sources may be in a
-# different format: nam.t{hh}z.prepbufr.tm{prevhour}.nr, where hh is 
-# either 00, 06, 12, or 18, and prevhour is the number of hours prior to
-# hh (00 through 05). If using custom staged data, you will have to
-# rename the files accordingly.
-# 
-# If data is retrieved from HPSS, it will automatically staged by this
-# this script.
+# Make sure the obs type is valid.  Then call a python script to check
+# for the presence of obs files on disk and get them if needed.
 #
+#-----------------------------------------------------------------------
 #
-# NOHRSC  snow accumulation observations
-# ----------
-# If data is available on disk, it must be in the following 
-# directory structure and file name conventions expected by verification
-# tasks:
+valid_obtypes=("CCPA" "MRMS" "NDAS" "NOHRSC")
+if [[ ! ${valid_obtypes[@]} =~ ${OBTYPE} ]]; then
+  print_err_msg_exit "\
+Invalid observation type (OBTYPE) specified for script:
+  OBTYPE = \"${OBTYPE}\"
+Valid observation types are:
+  $(printf "\"%s\" " ${valid_obtypes[@]})
+"
+fi
+
+script_bn="get_obs"
+cmd="\
+python3 -u ${USHdir}/${script_bn}.py \
+--var_defns_path "${GLOBAL_VAR_DEFNS_FP}" \
+--obtype ${OBTYPE} \
+--obs_day ${PDY}"
+print_info_msg "
+CALLING: ${cmd}"
+${cmd} || print_err_msg_exit "Error calling ${script_bn}.py."
```

Comment:
Why obfuscate the script name here? It makes searching hard.
```suggestion
cmd="\
python3 -u ${USHdir}/get_obs.py \
--var_defns_path "${GLOBAL_VAR_DEFNS_FP}" \
--obtype ${OBTYPE} \
--obs_day ${PDY}"
print_info_msg "
CALLING: ${cmd}"
${cmd} || print_err_msg_exit "Error calling get_obs.py."
```

Reply:
This is left over from when I was gradually transitioning from using the shell script to the python script for different types of obs.  Will fix.